playwright/test/view-items.test.ts

```diff
@@ -35,11 +47,38 @@ describe('Scenario 1: A user wants a large-scale view of an item', () => {
   });
 });
 
+describe('Scenario 4: A user wants to know how they can make use of an item', () => {
+  test('license information should be available', async () => {
+    await itemWithSearchAndStructures();
+    await page.click(`${accordionItemLicense} button`);
+    await page.waitForSelector(
+      `css=${accordionItemLicense} strong >> text="License:"`
```

Comment:
Why not just the text?

Reply:
If it were just text it could be anything on the page right? So, for example, if the contents included 'License:'

Reply:
No, but the content of the Contents accordion and the OCR text _could_ contain that string, although if it's not there now I guess it's a very remote possibility it ever will

Reply:
Yea, part of testing for things that a person will see / interact with is that you don't have to think of every eventuality. The implementation might deal with loads of things, but the interaction should be simple.